database/migrations/2016_02_27_103851_create_osu_slack_users_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOsuSlackUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('osu_slack_users', function (Blueprint $table) {
+            $table->string('slack_id', 50);
+            $table->integer('user_id');
```

Comment:
What should we do here? I assume I should add another migration to change the type on osu_slack_users.user_id to mediumint, but due to some bug the migration fails.